test/collection_test.js

```diff
@@ -174,6 +174,55 @@ describe("Collection", () => {
     });
   });
 
+  describe("#_setPlugins", () => {
+    var articles;
+
+    beforeEach(() => articles = testCollection());
+
+    it("should check that first argument is a supported type", () => {
+      try {
+        articles._setPlugins("bogus", {});
+        expect(false, "Should not reach here.");
+      } catch (err) {
+        expect(articles._plugins.remoteTransformers).to.be.instanceOf(Array);
+        expect(articles._plugins.remoteTransformers.length).to.equal(0);
+        expect(err.message).to.equal("Plugin type bogus not supported");
+      }
+    });
+
+    it("should set this._plugins.remoteTransformers", () => {
+      articles._setPlugins("remoteTransformers",
+          [ new QuestionMarkTransformer() ]);
+      expect(articles._plugins.remoteTransformers).to.be.instanceOf(Array);
```

Comment:
Oh, I would have expected that to fail because `typeof(new Array()) == "object"`, but I just tried it and you're right, in chai an array is an array and not an object. :)